tests/acceptance/features/apiSearch1/search.feature

```diff
@@ -291,3 +291,127 @@ Feature: Search
+      | shareType       | user          |
+      | permissionsRole | Viewer        |
+    And user "Alice" has a share "BrianPersonal" synced
+    When user "Alice" searches for "*inside*" inside folder "folderMain" in space "project101" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result should contain "1" entries
+    And the search result of user "Alice" should contain only these entries:
+      | folderMain/SubFolder1/subFOLDER2/insideTheFolder.txt |
+    But the search result of user "Alice" should not contain these entries:
+      | AlicePersonal/insideAlicePersonal.txt |
+      | BrianPersonal/insideBrianPersonal.txt |
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
+
+
+  Scenario Outline: search a file within Shares drive
+    Given using <dav-path-version> DAV path
+    And user "Alice" has created a folder "AlicePersonal" in space "Personal"
+    And user "Alice" has uploaded a file inside space "Personal" with content "inside Alice personal space" to "AlicePersonal/insideAlicePersonal.txt"
+    And user "Brian" has created a folder "BrianPersonal" in space "Personal"
+    And user "Brian" has uploaded file with content "inside Brian personal space" to "BrianPersonal/insideBrianPersonal.txt"
+    And user "Brian" has sent the following resource share invitation:
+      | resource        | BrianPersonal |
+      | space           | Personal      |
+      | sharee          | Alice         |
+      | shareType       | user          |
+      | permissionsRole | Viewer        |
+    And user "Alice" has a share "BrianPersonal" synced
+    When user "Alice" searches for "*inside*" inside folder "BrianPersonal" in space "Shares" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result should contain "1" entries
+    And the search result of user "Alice" should contain only these entries:
+      | BrianPersonal/insideBrianPersonal.txt |
+    But the search result of user "Alice" should not contain these entries:
+      | AlicePersonal/insideAlicePersonal.txt                |
+      | folderMain/SubFolder1/subFOLDER2/insideTheFolder.txt |
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
+
+  @issue-7060
+  Scenario Outline: check href value while searching for a file for each dav-path-version
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file with content "hello world inside root" to "file.txt"
+    When user "Alice" searches for "file.txt" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the value of the item "//d:response/d:href" in the response about user "Alice" should be "/remote.php/dav/spaces/%spaceid%/file.txt"
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
```

Comment:
```suggestion
      | old              | /remote.php/webdav/file.txt |
      | new              | /remote.php/dav/files/%user%/file.txt |
      | spaces           | /remote.php/dav/spaces/%spaceid%/file.txt |
```

Reply:
Since this is test coverage for bug report, I think it would be better to put in a separate PR.